spec/yaml/swiftnav/sbp/piksi.yaml

```diff
@@ -484,6 +484,45 @@ definitions:
             - 0:
                 desc: IFF_UP - interface is up
 
+ - NetworkUsage:
+    short_desc: Bandwidth usage measurement for a single interface.
+    desc: |
+      The bandwidth usage for each interface can be reported
+      within this struct and utilize multiple fields to fully
+      specify the type of traffic that is being tracked. As
+      either the interval of collection or the collection time
+      may vary, both a timestamp and period field is provided,
+      though may not necessarily be populated with a value. 
+    fields:
+      - duration:
+          type: u64
+          units: ms
+          desc: Duration over which the measurement was collected
+      - total_bytes:
```

Comment:
Does total bytes just equal tx_bytes + rx_bytes?

Reply:
Agree if we can commit to always having tx/rx split out. It seems reasonable but in a generic case we may not have access to tx/rx separately? The intent was for total to be the metric, and tx/rx and option added level of detail.